src/elements/List/ListItem.js

```diff
@@ -21,31 +20,21 @@ export default class ListItem extends Component {
   };
 
   render() {
-    const children = _.clone(this.props.children)
-    const classes = classNames('sd-list-item', this.props.className, 'item')
-    const description = _.clone(this.props.description)
-    const hasHeader = !!this.props.header
-    const header = <div className='header'>{this.props.header}</div>
-    const icon = _.clone(this.props.icon)
-    const image = _.clone(this.props.image)
+    const { children, className, description, header, icon, image, ...rest } = this.props
+    const classes = cx('sd-list-item', className, 'item')
 
-    const props = _.clone(this.props)
-    delete props.children
-    delete props.className
-    delete props.description
-    delete props.header
-    delete props.icon
-    delete props.image
+    const content = !header ? description : (
+      <div className='content'>
+        <div className='header'>{header}</div>
+        {description && <div className='description'>{description}</div>}
+      </div>
```

Comment:
Yes sir, otherwise the content doesn't align in the middle with `middle aligned content`.  The description div has a top margin which pushes it down a bit.